src/main/java/dev/j3fftw/soundmuffler/SoundMufflerListener.java

```diff
@@ -18,36 +19,36 @@
 
     public SoundMufflerListener(Plugin plugin) {
         super(plugin, ListenerPriority.NORMAL,
-            PacketType.Play.Server.NAMED_SOUND_EFFECT, PacketType.Play.Server.ENTITY_SOUND
+                PacketType.Play.Server.NAMED_SOUND_EFFECT, PacketType.Play.Server.ENTITY_SOUND
         );
     }
 
     @Override
     public void onPacketSending(PacketEvent event) {
         if (event.getPacketType() == PacketType.Play.Server.NAMED_SOUND_EFFECT
-            || event.getPacketType() == PacketType.Play.Server.ENTITY_SOUND
+                || event.getPacketType() == PacketType.Play.Server.ENTITY_SOUND
         ) {
             Location loc;
-            if (event.getPacketType() == PacketType.Play.Server.NAMED_SOUND_EFFECT){
+            if (event.getPacketType() == PacketType.Play.Server.NAMED_SOUND_EFFECT) {
                 int x = event.getPacket().getIntegers().read(0) >> 3;
                 int y = event.getPacket().getIntegers().read(1) >> 3;
                 int z = event.getPacket().getIntegers().read(2) >> 3;
                 loc = new Location(event.getPlayer().getWorld(), x, y, z);
             } else if (event.getPacketType() == PacketType.Play.Server.ENTITY_SOUND) {
                 loc = event.getPlayer().getWorld().getEntities().stream()
-                    .filter(e -> e.getEntityId() == event.getPacket().getIntegers().read(0))
-                    .map(Entity::getLocation)
-                    .findAny().orElse(null);
+                        .filter(e -> e.getEntityId() == event.getPacket().getIntegers().read(0))
+                        .map(Entity::getLocation)
+                        .findAny().orElse(null);
```

Comment:
Remove the unnecessary formatting changes. Seems your syntax IDE settings slightly differ here. Just remove all the formatting stuff